includes/resourceloader/ResourceLoaderUserModule.php

```diff
@@ -37,6 +37,12 @@ protected function getPages( ResourceLoaderContext $context ) {
 			// Get the normalized title of the user's user page
 			$username = $context->getUser();
```

Comment:
`$context->getUser()` return the unchecked value of the "user" GET parameter in the URL. Anyone can craft a request that would be an invalid title in Mediawiki world. However this can also come from some bad escaping thereof or a bug in `makeTitleSafe`. Let's then find the root cause... :)